workers/unity/Packages/io.improbable.gdk.core/View/DiffComponentStorage.cs

```diff
@@ -0,0 +1,143 @@
+using System;
+using System.Collections.Generic;
+using Improbable.Worker.CInterop;
+
+namespace Improbable.Gdk.Core
+{
+    public abstract class DiffComponentStorage<TUpdate> : IDiffUpdateStorage<TUpdate>, IDiffComponentAddedStorage<TUpdate>, IDiffAuthorityStorage
+        where TUpdate : ISpatialComponentUpdate
+    {
+        private readonly HashSet<EntityId> entitiesUpdated = new HashSet<EntityId>();
```

Comment:
Can you make this protected since I'll need that in #1298 ?

Reply:
👍